crates/networking/rpc/types/payload.rs

```diff
@@ -35,10 +35,19 @@ pub struct ExecutionPayloadV3 {
     pub block_hash: H256,
     transactions: Vec<EncodedTransaction>,
     withdrawals: Vec<Withdrawal>,
-    #[serde(with = "serde_utils::u64::hex_str")]
-    blob_gas_used: u64,
-    #[serde(with = "serde_utils::u64::hex_str")]
-    excess_blob_gas: u64,
+    // ExecutionPayloadV3 fields. Optional since we support V2 too
+    #[serde(
+        skip_serializing_if = "Option::is_none",
+        with = "serde_utils::u64::hex_str_opt",
+        default = "Option::default"
+    )]
+    pub blob_gas_used: Option<u64>,
+    #[serde(
+        skip_serializing_if = "Option::is_none",
+        with = "serde_utils::u64::hex_str_opt",
+        default = "Option::default"
```

Comment:
I think we can just use `default` here, we don't need to explicitly set the default value